content/07.pathogenesis.md

```diff
@@ -118,7 +118,7 @@ Coronaviruses such as SARS-CoV-1 are also able to evade the humoral immune respo
 
 #### Host Cell Susceptibility
 
-ACE2 and TMPRSS2 have been identified as the primary entry portal and as a critical protease, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
+ACE2 and TMPRSS2 have been identified as the primary entry portals and as a critical proteases, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
```

Comment:
```suggestion
ACE2 and TMPRSS2 have been identified as the primary entry portal and as a critical protease, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
```
ACE2 maps to entry portal and TMPRSS2 maps to protease, so singular is correct here!